GVFS/GVFS.Service/IRepoMounter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using GVFS.Common.Tracing;
+
+namespace GVFS.Service
+{
+    public interface IRepoMounter
+    {
+        bool MountRepository(string repoRoot, int sessionId, ITracer tracer);
+        string GetUserId(int sessionId);

```

Comment:
Does this method fit with a type that is about mounting repositories?

Reply:
As discussed moved it to `GVFSPlatform`. Renamed the method to `GetUserIdFromSessionId()`